Deploy/src/lambda.tf

```
@@ -0,0 +1,35 @@
+data "aws_iam_policy" "AWSLambdaBasicExecutionRole" {
+  arn = "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
+}
+
+data "aws_iam_policy_document" "hello_world" {
+  statement {
+    principals {
+      type        = "Service"
+      identifiers = ["lambda.amazonaws.com"]
+    }
+    effect  = "Allow"
+    actions = ["sts:AssumeRole"]
+  }
+}
+
+data "archive_file" "hello_world_lambda" {
+  type        = "zip"
+  source_file = "../functions/index.js"
+  output_path = var.artifacts_output_path
+}
+
+resource "aws_lambda_function" "hello_world_lambda" {
+  filename         = var.artifacts_output_path
+  function_name    = "hello_world_lambda_function"

```

No point spelling out the obvious
```suggestion
  function_name    = "hello_world"
```